src/components/Search.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { FaSearch } from 'react-icons/fa';
+import '../styles/search.css';
+
+const Search = () => (
+  <div className="searchBar">
+    <input type="text" placeholder="Search..." />
+    <FaSearch className="search" />
+  </div>
+);
```

Comment:
- [ ] Good job on the design, but this part should work since you are using the search, you can call as  example. category name. to make it work.

Reply:
sure I added it